client/dist/style.css

```diff
@@ -2,10 +2,8 @@
   border: 1px solid #e4e4e4;
   width: 376px;
   height: 462px;
-  /* height: 611px; */
   position: static;
-  font-family: Circular, -apple-system, system-ui, 'Raleway', 'Montserrat', 'Roboto', 'Helvetica Neue', sans-serif;
-  /* font-family: 'Montserrat', 'Roboto', 'Open Sans', Arial; */
```

Comment:
So the only thing I can say for the whole of the CSS page is that it could be a good idea to add properties in alphabetical order, this is mostly for readability.

There is some commented out code, not sure if you want to keep that for testing purposes?